perftracker/views.py

```diff
@@ -158,6 +162,93 @@ def pt_comparison_tables_info_json(request, api_ver, project_id, cmp_id, group_i
     return JsonResponse(ret, safe=False)
 
 
+@csrf_exempt
+def pt_comparison_section_properties_save(request, api_ver, project_id, cmp_id, group_id, section_id):
+    try:
+        data = json.loads(request.body.decode("utf-8"))
+    except json.JSONDecodeError:
+        return HttpResponseBadRequest("Wrong json")
+
+    data = data[str(section_id)]
+    PTComparisonChartTrainDataModel.pt_validate_json(data)
+
+    formatted_data = '|'.join([f'{x};{y}' for x, y in data['data']])
+    incline = ComparisonModel._pt_get_type(CMP_CHART_INCLINES, data, 'incline')
+
+    record = PTComparisonChartTrainDataModel(data=formatted_data, incline=incline)
+    record.save()
+
+    return JsonResponse(data, safe=False)
+
+
+@csrf_exempt
+def pt_comparison_analyze_json(request, api_ver, project_id, cmp_id):
+
+    try:
+        data = json.loads(request.body.decode("utf-8"))
+    except json.JSONDecodeError:
+        return HttpResponseBadRequest("Wrong json")
+
+    round_to = 3
+    mse_categories = ('low', 'med', 'high')
+    mse_categories_amount = len(mse_categories)
+    mse_values = []
+    rv = {}
+
+    # coefficient belongs to (0, 1) for little outliers detection
+    # coefficient belongs to (1, +inf) for rough outliers detection
+    # default value is 1.
+    outlier_coefficient = 1.
+
+    for graph_id, graph_data in data.items():
```

Comment:
pls move all business logic to separate file, remove dependency from numpy, make sure site can run w/o this functionality